environment.yml

```
@@ -22,8 +22,8 @@ dependencies:
   - bottleneck
   # remember to match xclim version in environment-docs.yml as well
   - xclim == 0.31
-  - clisops >=0.6.4
+  - clisops >= 0.8.0
   - pywps ==4.4.5
   - parse
-  - xesmf >=0.5.3,!=0.6.1  # clisops 0.7 bugs with xESMF 0.6.1
-  - ESMpy <= 8.1.1 # Bug in ESMF 8.2 with spatial averaging.
\ No newline at end of file
+  - xesmf == 0.6.2  # clisops 0.7 bugs with xESMF 0.6.1
```

```suggestion
  - xesmf >= 0.6.2
```
So we are allowed an update of xesmf before updating finch? And to replicate the requirements file.